docs/geode-native-docs/security/sslclientserver.html.md.erb

```diff
@@ -42,17 +42,17 @@ To install OpenSSL:
 2. Extract the archive in a directory of your choice. For example:
 
     ```
-    $ tar xvzf openssl-1.0.1u.tar.gz
-    x openssl-1.0.1u/ACKNOWLEDGMENTS
-    x openssl-1.0.1u/apps/
-    x openssl-1.0.1u/apps/app_rand.c
+    $ tar xvzf openssl-1.0.2l.tar.gz
```

Comment:
Leave off the version numbers for all of these examples. These bits of example on OpenSSL setup should be in a separate doc.